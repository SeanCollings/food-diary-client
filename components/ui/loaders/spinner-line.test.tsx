import { render } from '@testing-library/react';
import { SpinnerLine } from './';

describe('<SpinnerLine>', () => {
  it('should render', () => {
    const { asFragment } = render(<SpinnerLine />);
    expect(asFragment()).toMatchSnapshot();
  });
});
