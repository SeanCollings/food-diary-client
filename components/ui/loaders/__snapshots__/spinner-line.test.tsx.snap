// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpinnerLine> should render 1`] = `
<DocumentFragment>
  <div
    class="sc-b37157e9-0 hyBKQb"
  >
    <div
      class="sc-b37157e9-1 hTsLLc lds-ring"
      color="#ffffff"
      height="60"
      opacity="1"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
